src/register/messages.jsx

```diff
@@ -33,6 +33,11 @@ const messages = defineMessages({
     description: 'Placeholder for the country options dropdown.',
   },
   // Help text
+  'help.text.name': {
+    id: 'help.text.name',
+    defaultMessage: 'This name will be used by any certificates that you earn.',
```

Comment:
The original message on the ticket does not have a full stop (.) at the end `This name will be used by any certificates that you earn`

Reply:
yeah I noticed that but other help text messages do have full stop that is why I have added it. Will merge and move this to review and see if Jon needs me to update it.